src/unittest/test.cpp

```diff
@@ -249,7 +252,12 @@ bool run_tests()
 		<< num_total_tests_run << " failed individual tests)." << std::endl
 		<< "    Testing took " << tdiff << "ms total." << std::endl
 		<< "++++++++++++++++++++++++++++++++++++++++"
-		<< "++++++++++++++++++++++++++++++++++++++++" << std::endl;
+		<< "++++++++++++++++++++++++++++++++++++++++" << std::endl
+	        << "Catch test results: " << std::endl;
+    auto catch_test_failures = Catch::Session().run();
```

Comment:
this just runs the other tests unconditionally after the normal ones with no integration with `--test-module`